chia/rpc/data_layer_rpc_api.py

```diff
@@ -4,6 +4,9 @@
 from pathlib import Path
 from typing import TYPE_CHECKING, Any, ClassVar, Optional, Union, cast
 
+# todo input assertions for all rpc's
```

Comment:
comment from original commit.  doesn't really have anything to do with anything here.

```suggestion
```